src/avram/upsert.cr

```diff
@@ -0,0 +1,32 @@
+module Avram::Upsert
+  macro upsert_lookup_columns(*attribute_names)
```

Comment:
Out of curiosity, what does the error look like if you misspell a column name, or add a name here that doesn't exist? `upset_lookup_columns :bloopity`

Reply:
Pretty good actually! Shows the macro that called it and gives a nice suggestion (assuming it is a typo and not totally wrong)

<img width="861" alt="Screen Shot 2021-07-12 at 2 10 33 PM" src="https://user-images.githubusercontent.com/22394/125335765-eecd0300-e31a-11eb-9e28-0fe0c4f9d9f3.png">

Reply:
boom! That's way nice 👍